README.md

```diff
@@ -317,6 +317,35 @@ export default withStyles(({ color, unit }) => ({
 
 `className` and `style` props must not be used on the same elements as `css()`.
 
+### Using the value of a prop / high cardinality
+
+To include styles that use the value of a prop (or any other styles that have a high cardinality and shouldn't be in a themed stylesheet), you can use inline styles in conjunction with `withstyles()`. 
+
+```jsx
+function MyComponent({ bold, padding, styles }) {
+  const {
+    backgroundImageUrl,
+  } = this.props;
```

Comment:
Since this is a stateless functional component, `this.props` is not right. This should be destructured in the method signature along with the rest of the props.